server/sandbox_network.go

```diff
@@ -230,6 +230,7 @@ func (s *Server) newPodNetwork(ctx context.Context, sb *sandbox.Sandbox) (ocicni
 	}
 
 	network := s.config.CNIPlugin().GetDefaultNetworkName()
+	podAnnotations := sb.Annotations()
```

Comment:
could you please make this change for 'newPodNetwork' and 'networkStop' functions? for cmdAdd and cmdDel.

thanks @LenkaSeg